docs/reference/builder.md

```diff
@@ -47,7 +47,7 @@ performance, exclude files and directories by adding a `.dockerignore` file to
 the context directory.  For information about how to [create a `.dockerignore`
 file](#dockerignore-file) see the documentation on this page.
 
-Traditionally, the `Dockerfile` is called `Dockerfile` and located in the root
+Traditionally, the building file is called `Dockerfile` and located in the root
```

Comment:
I'm a bit unsure if we should use "building file" here, as that would be introducing a new concept. I can see that the current sentence looks a bit funny, but it _is_ correct.